.travis.yml

```diff
@@ -1,52 +1,50 @@
 language: bash
-
 sudo: required
-services: docker 
-
+services: docker
 env:
   global:
-    - DOCKER_APP_VERSION=17.05.0~ce-0~ubuntu-trusty
-
+  - DOCKER_APP_VERSION=17.05.0~ce-0~ubuntu-trusty
```

Comment:
you actually wanna upgrade that some time ?
to xenial or once they have all the packages even bionic ?

Reply:
we should still be good without that for now